analyses/cell-type-scimilarity/setup-analysis.sh

```diff
@@ -20,28 +25,45 @@ cd ${module_dir}
 scripts_dir="scripts"
 ref_dir="references"
 
+# Define argument defaults
+# default is to grab the annotation only model from s3, otherwise the full model from zenodo can be downloaded 
+zenodo=${zenodo:-0}
+
 ###################################################################
 ################## Download SCimilarity model #####################
 ###################################################################
 
-# define the url to the model
-scimilarity_model_url="https://zenodo.org/records/10685499/files/model_v1.1.tar.gz?download=1"
-
 # output model directory
 model_dir="models"
 mkdir -p $model_dir
 
 # compressed output file and final model directory
-compressed_scimilarity_model="${model_dir}/model_v1.1.tar.gz"
 scimilarity_model_dir="${model_dir}/model_v1.1"
 
-if [[ ! -d $scimilarity_model_dir ]]; then
+if [[ ! -d $scimilarity_model_dir && $zenodo -eq 0 ]]; then
+
+  echo "Downloading only the annotation SCimilarity model from s3"
+  s3_model='s3://scpca-references/celltype/scimilarity_references/model_v1.1'
+  aws s3 cp $s3_model $scimilarity_model_dir --exclude "cellsearch/*" --recursive --no-sign-request
```

Comment:
Just to clean up the GHA output a bit.
```suggestion
  aws s3 cp $s3_model $scimilarity_model_dir --exclude "cellsearch/*" --recursive --no-sign-request --no-progress
```